lib/genome/importers/dtc/dtc.rb

```diff
@@ -55,10 +55,10 @@ def create_interaction_claims
         gene_name.split(',').each do |indv_gene|
           gene_claim = create_gene_claim(indv_gene, 'DTC Gene Name')
           interaction_claim = create_interaction_claim(gene_claim, drug_claim)
-          unless pmid.nil? || pmid == '""' || pmid == "''" || pmid[0] =='-' || pmid =='15288657'
-            create_interaction_claim_publication(interaction_claim, row['pubmed_id'])
+          unless pmid.nil? || pmid == "" || pmid == '""' || pmid == "''" || pmid[0] =='-' || pmid =='15288657'
+            create_interaction_claim_publication(interaction_claim, pmid)
           end
-          unless mechanism.nil?
+          unless mechanism.nil? || mechanism == ""
```

Comment:
You could make this `unless mechanism.blank?` which with check for `nil`, whitespace only, or empty string, but it will work just fine as written. Just a note for the future.